test/integration/css-client-nav/test/index.test.js

```diff
@@ -180,6 +181,8 @@ describe('CSS Module client-side navigation', () => {
     })
     afterAll(async () => {
       proxyServer.close()
+      // something is hanging onto the process, so we need to SIGKILL
+      await killProcess(app.pid, 'SIGKILL')
```

Comment:
I wasn't able to track down why the app wouldn't close. I tried closing `proxyServer` as well as `proxy` in different orders, but both of them would hang if I waited for the `close` callback. I tried killing both of those with `killApp` which seemed to shut them down, but `killApp(app)` would still hang.

Using `SIGKILL` was the only way I found to work. Before this change, it would have been calling `process.exit(0)` immediately on receiving SIGTERM, so the end result should be the same.